### ISSUE TITLE
build error : lvmDriver does not implement volume.Driver (missing Capabilities method)

### ISSUE THREAD

**Issue by monkeydri** [projectatomic/docker-lvm-plugin#13]
After cloning repo and trying to `make` I have the following error :

```
monkeydri@server:~/docker-lvm-plugin$ make
go-md2man -in man/docker-lvm-plugin.8.md -out docker-lvm-plugin.8
/usr/bin/go build -o docker-lvm-plugin .
# _/home/monkeydri/docker-lvm-plugin
./main.go:65: cannot use lvm (type *lvmDriver) as type volume.Driver in argument to volume.NewHandler:
	*lvmDriver does not implement volume.Driver (missing Capabilities method)
Makefile:19: recipe for target 'lvm-plugin-build' failed
make: *** [lvm-plugin-build] Error 2
```

go-plugins-helpers/volume : latest version (commit f3a3876)
go version : 1.6
Docker version 1.10.3, build 20f81dd
os : ubuntu 16.04.1

This error looks similar to the error described here : https://github.com/calavera/docker-volume-glusterfs/pull/33 for project docker-volume-glusterfs :

```
src/github.com/calavera/docker-volume-glusterfs/main.go:38: cannot use d (type glusterfsDriver) as type volume.Driver in argument to volume.NewHandler:
glusterfsDriver does not implement volume.Driver (missing Capabilities method)
```

which was fixed by commit https://github.com/calavera/docker-volume-glusterfs/pull/33/commits/5a818b221fd0d4ad7a452945cbeda1855b82fbd1

**Comment by monkeydri** [comment]
`Capabilities` where added by commit https://github.com/docker/go-plugins-helpers/pull/51/commits/7f7e1f8cba8ccb022b28b400cbefa60452eb4728

**Comment by shishir-a412ed** [comment]
Fixed with #15
